app/models/movie.rb

```diff
@@ -1,5 +1,3 @@
 class Movie < ApplicationRecord
-  validates :genre, presence: true
-  validates :title, presence: true
-  validates :url, presence: true
+  validates :title, :url, presence: true
```

Comment:
`genre` も必須にしなければなりません。

```suggestion
  validates :genre, :title, :url, presence: true
```